ppx/browser/ppx_deriving_json_js.ml

```diff
@@ -190,10 +190,19 @@ module To_json = struct
     let loc = t.rcd_loc in
     let fs =
       List.map2 t.rcd_fields es ~f:(fun ld x ->
-          let n = ld.pld_name in
-          let n = Option.value ~default:n (ld_attr_json_key ld) in
-          let this = derive ld.pld_type x in
-          map_loc lident n, this)
+          let k = ld.pld_name in
+          let k = Option.value ~default:k (ld_attr_json_key ld) in
```

Comment:
```suggestion
          let k =
            let k = ld.pld_name in
            Option.value ~default:k (ld_attr_json_key ld) in
```

Reply:
I kinda like this pattern, could also use it for `v`, but this is optional if you don't like the extra nesting